aadhaar-ocr/src/main/java/com/sayukth/aadhaar_ocr/ui/DetectAadhaarPresenter.java

```diff
@@ -61,31 +102,48 @@ public void getImageDataAsText(Bitmap photo) {
             stringBuilder.append("#" + imageText + "#");
             stringBuilder.append("\n");
 
+            boolean isFrontMatch = imageText.contains(DOB) || imageText.contains(Year) || imageText.contains(OF) || imageText.contains(Birth);
```

Comment:
DEFINE fields in class level

private static final String DOB_KEYWORD = "DOB";
private static final String YEAR_KEYWORD = "Year";
private static final String BIRTH_KEYWORD = "Birth";
private static final String ADDRESS_KEYWORD = "Address";


```suggestion
            boolean isFrontMatch = imageText.contains(DOB_KEYWORD) || imageText.contains(YEAR_KEYWORD) || 
                       imageText.contains("OF") || imageText.contains(BIRTH_KEYWORD);

```